AUTHORS.rst

```diff
@@ -64,31 +65,30 @@ Aviv Salem
      PLAIN authentication support
 
 Bug Reports
-------------
-Brian Jackson
-Neil Martinsen-Burrell
-Mark Eichin (eichin _AT_ thok.org)
-Fergal Daly (fergal _AT_ esatclear.ie)
-Bob Yexley (bob _AT_ yexley.net)
-Rhett Garber
-Erik Quaeghebeur
-Benjamin Morrise
-Brandon Rhodes
-Thomi Richards
-Ivan Zderadicka
-Raffael Vogler
-
-External Projects
------------------
+~~~~~~~~~~~
+- Brian Jackson
+- Neil Martinsen-Burrell
+- Mark Eichin (eichin _AT_ thok.org)
+- Fergal Daly (fergal _AT_ esatclear.ie)
+- Bob Yexley (bob _AT_ yexley.net)
+- Rhett Garber
+- Erik Quaeghebeur
+- Benjamin Morrise
+- Brandon Rhodes
+- Thomi Richards
+- Ivan Zderadicka
+- Raffael Vogler
+
+
+Other Contributions
+~~~~~~~~~~~~~~~~~~~
 IMAPClient uses Michael's excellent 'Mock' package.
```

Comment:
+1. Let's:
* remove the specific mention of Mock
* remove the specifics of "who did what"
* remove the bug reports section